### PR TITLE
Properly prefixed command ids ledger api test tool tests [KVL-658]

### PR DESCRIPTION
**Pull request by kamil-da**
Removed ad hoc usages of `UUID` in `ledger-api-test-tool` for id generation for commands and keys.
This change should facilitate debugging.

CHANGELOG_BEGIN
CHANGELOG_END

### Pull Request Checklist

- [x] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [x] Include appropriate tests
- [x] Set a descriptive title and thorough description
- [x] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [x] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [x] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/participant/ParticipantTestContext.scala [hunk 1]
```diff
@@ -114,15 +114,14 @@ private[testtool] final class ParticipantTestContext private[participant] (
     */
   val end: LedgerOffset =
     LedgerOffset(LedgerOffset.Value.Boundary(LedgerOffset.LedgerBoundary.LEDGER_END))
-
-  private[this] val identifierPrefix = s"$applicationId-$endpointId-$identifierSuffix"
```
Comment (SamirTalwar-DA) [inline ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/participant/ParticipantTestContext.scala:118]: I think we could still keep this.

**Review by SamirTalwar-DA — APPROVED** [review]
Looks good to me. Have you verified that the logs are easier to parse now?

**Comment by kamil-da** [comment]
I did check that command IDs and key IDs are now properly prefixed. Searching by test case name shows now some more useful results.

**Comment by SamirTalwar-DA** [comment]
Sounds good.

You need to run `./fmt.sh --diff`.

File: ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/participant/ParticipantTestContext.scala (hunk 1 above)
Comment (kamil-da) [reply ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/participant/ParticipantTestContext.scala:118]: Restored to the initial state 👍

**Pull request merged**